sympy/polys/rings.py

```diff
@@ -1653,7 +1749,7 @@ def subs(self, *args, **kwargs):
         else:
             return self._subs(subs_dict)
 
-    def prem(self, g, x=None):
+    def prem(self, g: PolyElement[Er], x: PolyElement[Er] | int | None=None) -> PolyElement[Er]:
```

Comment:
When there are a type annotation for a parameter that ha a default value it reads better to have spaces around the `=` like `None = None` because `|` binds more tightly than the `=` in this case.